server/src/infra/repositories/person.repository.ts

```diff
@@ -10,6 +10,45 @@ export class PersonRepository implements IPersonRepository {
     @InjectRepository(AssetFaceEntity) private assetFaceRepository: Repository<AssetFaceEntity>,
   ) {}
 
+  async updateAssetsId(primaryPersonId: string, mergePersonId: string): Promise<number> {
+    const result = await this.assetFaceRepository
+      .createQueryBuilder()
+      .update()
+      .set({ personId: primaryPersonId })
+      .where({ personId: mergePersonId })
+      .execute();
+
+    return result.affected ?? 0;
+  }
+
+  /**
+   * Delete the assets that containers faces from both persons.
+   * Normally, this shouldn't happen, however there are some edge cases, where your photo is taken
+   * with an older version of your self, and the face detection algorithm detects both faces as different.
+   * @param ids Array of personId.
+   * @returns sAn array of assetId that contains faces from both persons.
+   */
+  async deleteIdenticalAssets(primaryPersonId: string, mergePersonId: string): Promise<string[]> {
```

Comment:
We're not deleting assets, we are deleting faces. Face is just a link between people and assets. You could say we are deleting the person A _faces_ that also have person B in them already, in preparation for merging person B into person A. Maybe this would be better named:

```
deleteFacesForSharedAssets(oldPersonId: string, newPersonId: string)
```